src/plugins/gui/amountQuotePlugin.ts

```diff
@@ -479,7 +480,10 @@ export const amountQuoteFiatPlugin: FiatPluginFactory = async (params: FiatPlugi
           }
 
           // Restrict light accounts from buying more than $50 at a time
-          if (isLightAccount && gt(bestQuote.fiatAmount, '50')) {
+          if (
+            isLightAccount &&
```

Comment:
Yeah was trying to steer away from wrapping an if inside of an if to avoid useless calls for non light accounts but I'll make this cleaner to read instead